mam/v2/test_utils/test_utils.c

```diff
@@ -696,12 +689,12 @@ static void test_delete_sponge(ialloc *a, sponge_t *s) {
 //   test_mss _m[1];
 //   test_ntru_t _n[1];
 //
-//   sponge_t *s = test_sponge_init(_s);
-//   spongos_t *sg = test_spongos_init(_sg, s);
-//   prng_t *p = test_prng_init(_p, s);
-//   prng_t *pa = test_prng_init(_pa, s);
-//   prng_t *pb = test_prng_init(_pb, s);
-//   wots_t *w = test_wots_init(_w, s);
+//   sponge_t *spongos = test_sponge_init(_s);
```

Comment:
Why did these dead code be kept?

Reply:
because we import it iteratively,eventually, it will be either uncommented or removed